simulator.py

```diff
@@ -292,7 +292,7 @@ def run(self, city):
                 assert self.has_logged_symptoms is True # FIXME: assumption might not hold
 
             if self.is_infectious and self.env.timestamp - self.infection_timestamp >= datetime.timedelta(days=self.recovery_days):
-                if self.never_recovers or True: # re-infection assumed negligble
+                if self.never_recovers: # re-infection assumed negligble
```

Comment:
this assumes re-infection is possible. might be we can add a parameter - REINFECTION_PROBABILITY and use that as a parameter here in `if`?